src/current/v23.1/migration-overview.md

```diff
@@ -5,87 +5,304 @@ toc: true
+	{{site.data.alerts.callout_info}}
+	The Schema Conversion Tool accepts `.sql` files from PostgreSQL, MySQL, Oracle, and Microsoft SQL Server.
+	{{site.data.alerts.end}}
+
+- Alternatively, manually convert the schema according to our [schema design best practices](#schema-design-best-practices){% comment %}and data type mappings{% endcomment %}. You can also [export a partially converted schema](../cockroachcloud/migrations-page.html#export-the-schema) from the Schema Conversion Tool to finish the conversion manually.
+
+Then import the converted schema to a CockroachDB cluster:
+
+- For {{ site.data.products.serverless }}, use the Schema Conversion Tool to [migrate the converted schema to a new {{ site.data.products.serverless }} database](../cockroachcloud/migrations-page.html#migrate-the-schema).
+- For {{ site.data.products.core }} or {{ site.data.products.dedicated }}, pipe the [data definition language (DDL)](sql-statements.html#data-definition-statements) directly into [`cockroach sql`](cockroach-sql.html). You can [export a converted schema file](../cockroachcloud/migrations-page.html#export-the-schema) from the Schema Conversion Tool.
+	{{site.data.alerts.callout_success}}
+	For the fastest performance, you can use a [local, single-node CockroachDB cluster](cockroach-start-single-node.html#start-a-single-node-cluster) to convert your schema and [check the results of queries](#test-query-results-and-performance).
+	{{site.data.alerts.end}}
+
+#### Load test data
+
+After [converting the schema](#convert-the-schema), load a subset of your data onto CockroachDB so that you can [test your application queries](#validate-queries). Use the legacy database's tooling to extract the data to a `.sql` file. Then use one of the following methods to migrate the data:
+
+- {% include {{ page.version.version }}/migration/load-data-import-into.md %}
+- {% include {{ page.version.version }}/migration/load-data-third-party.md %} Within the tool, you can select a subset of database tables to migrate to the test cluster.
+- {% include {{ page.version.version }}/migration/load-data-copy-from.md %}
+
+#### Validate queries
+
+After you [load the test data](#load-test-data), validate your queries on CockroachDB. You can do this by [shadowing](#shadowing) or by [manually testing](#test-query-results-and-performance) the queries.
+
+##### Shadowing
+
+You can "shadow" your production workload by replicating all writes from the legacy database to CockroachDB. To do this, use a [change data capture (CDC)](cdc-queries.html) service such as Kafka. {% comment %}The CockroachDB Live Migration Service (MOLT LMS) can also write queries to both databases simultaneously.{% endcomment %} You can then [test the queries](#test-query-results-and-performance) on CockroachDB for consistency, performance, and potential issues with the migration.
```

Comment:
Same LMS private preview comments

Reply:
@ajstorm Since we're not mentioning LMS until Limited Access (per Shannon's comment on this PR), is there another CDC tool I can mention here? Or is it that CDC isn't the method typically used for shadowing (what is)?

Reply:
there isnt another tool. Essentiall you've written it well--use a tool to do dual writes or do it yourself from the app